ui/mirage/scenarios/topo.js

```diff
@@ -109,7 +109,3 @@ export function topoMedium(server) {
     forceRunningClientStatus: true,
   });
 }
-
-export function topoMediumBatch(server) {}
-
-export function topoMediumVariadic(server) {}
```

Comment:
There’s another empty `topoSmallProblems` higher up… not a big deal but might be worth removing. I’m viewing “changes since” or whatever hence me not commenting on that line 🙃